tutor/commands/upgrade/compose.py

```diff
@@ -165,6 +166,48 @@ def upgrade_from_quince(context: click.Context, config: Config) -> None:
     upgrade_mongodb(context, config, "6.0.14", "6.0")
     upgrade_mongodb(context, config, "7.0.7", "7.0")
 
+    if not config["RUN_MYSQL"]:
+        fmt.echo_info(
+            "You are not running MySQL (RUN_MYSQL=false). It is your "
+            "responsibility to upgrade your MySQL instance to v8.4. There is "
+            "nothing left to do to upgrade from Quince."
+        )
+        return
+
+    # Revert the MySQL image first to build data dictionary on v8.1
```

Comment:
hmmm, if someone has already ran upgrade from quince, they would not be able to run it again successfully due to Mongo upgrade. Should we convert this into a do command as well?

Reply:
Hmm, I didn't think of that. Maybe we can add a small check if Mongo is already updated to v7.0.7 and we should not upgrade it then?

Reply:
Hmmm, I think that having to run this as part of upgrade is going to be weird, considering it is mid release. 
We can add do command and maybe package this in upgrade for sumac?

Reply:
Alright, sure. This change shouldn't take too much time. I'll convert this into a do command and update the PR.

Reply:
I suspect this is to be removed?